signing/README.md

```diff
@@ -1,11 +1,44 @@
-# Demo of offline signing
+# Signing
+
+## Configuration of NitroKeys
+
+NitroKeys are a personal security token used to hold the signing keys for
+TEST and DEV devices.  NitroKeys can be used to sign tests and binaries for
```

Comment:
nit: We can consider making this description a bit more generic since we are also planning to support provisioning of ROM_EXT keys in NitroKeys. 

In the future this will also include tooling for keys generated as part of the ownership transfer flow.